stock_split_picking/readme/DESCRIPTION.rst

```diff
@@ -2,3 +2,9 @@ This module adds a "Split" button on the outgoing pickings form.
 
 It works like the classical picking Transfer but it leaves both pickings
 (picking and its backorder) as confirmed without processing the transfer.
+<<<<<<< HEAD
+=======
+
+You can also choose to put all moves into separate pickings, or select moves
+to be put into a new picking.
+>>>>>>> e7822196... [ADD] stock_split_picking: Mode selection to split in different ways
```

Comment:
Tested locally and looks good, yet there are some conflicts and issues with pre-commit. I'm now fixing them.